docs/platforms/javascript/common/options.mdx

```diff
@@ -0,0 +1,454 @@
+
+</SdkOption>
+
+<SdkOption name="beforeSendSpan" type='(span: SpanJSON) => SpanJSON | null'>
+
+<Include name="platforms/configuration/options/warn-before-send-span.mdx" />
+
+This function is called with a serialized span object, and can return a modified span object, or `null` to skip sending the span. This might be useful for manually stripping PII from spans or to remove individual spans from the span tree.
+This function is only called for child spans of the root span. If you want to modify or drop the root span, including all of its child spans, use [`beforeSendTransaction`](#beforeSendTransaction) instead.
+
+</SdkOption>
+
+<SdkOption name="beforeBreadcrumb" type='(breadcrumb: Breadcrumb, hint?: BreadcrumbHint) => Breadcrumb | null'>
+
+This function is called with a breadcrumb object before the breadcrumb is added to the scope. When nothing is returned from the function, the breadcrumb is dropped. To pass the breadcrumb through, return the first argument, which contains the breadcrumb object.
+The callback gets a second argument (called a "hint") which contains the original object from which the breadcrumb was created to further customize what the breadcrumb should look like.
+
+</SdkOption>
+
+## Tracing Options
+
+<SdkOption name="tracesSampleRate" type='number' envVar='SENTRY_TRACES_SAMPLE_RATE'>
+
+A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app. Either this or `tracesSampler` must be defined to enable tracing.
+
+</SdkOption>
+
+<SdkOption name="tracesSampler" type='(samplingContext: SamplingContext) => number | boolean'>
+
+A function responsible for determining the percentage chance a given transaction will be sent to Sentry. It will automatically be passed information about the transaction and the context in which it's being created, and must return a number between `0` (0% chance of being sent) and `1` (100% chance of being sent). Can also be used for filtering transactions, by returning 0 for those that are unwanted. Either this or `tracesSampleRate` must be defined to enable tracing.
+
+The `samplingContext` object passed to the function has the following properties:
+
+- `parentSampled`: The sampling decision of the parent transaction. This is `true` if the parent transaction was sampled, and `false` if it was not.
+- `name`: The name of the span as it was started.
+- `attributes`: The initial attributes of the span.
+  <PlatformCategorySection supported={["server", "serverless"]}>
+    - `normalizedRequest`: Request information of the currently active HTTP
+    server request, if applicable.
+  </PlatformCategorySection>
+
+</SdkOption>
+
+<SdkOption name="tracePropagationTargets" type='Array<string | RegExp>'>
```

Comment:
I'd also add a default value entry into the table. Which needs to be different on browser and server.

Reply:
yeah this is a bit tricky because 1. it is hard to implement this nicely (we can, if we want to) and especially 2. the default value here is not an actual value but computed (e.g. based on the current host), which is why I opted to rather explain the default in this case?

Reply:
idea (logaf-l) should we then add something liks "see description" for the default value field?